### PR TITLE
init RGBVector split and join nodes

### PR DESCRIPTION
**Pull request by woeishi**

### CODE REVIEW

**Comment by woeishi** [comment]
bump

File: vvvv45/addonpack/src/nodes/plugins/Color/RGBVector/RGBVectorNode.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			FOutPin.GetValuePointer(out outData);
+			
+			for (int i=0; i<inCount*4; i++)
+				outData[i] = inData[i];
+//			Memory.Copy(outData, inData, (uint)(inCount*4*sizeof(double)));
+		}
+		
+		#region PluginInfo
+		[PluginInfo(Name = "RGB", Category = "Color", Version = "Join Vector", Help = "creates a colour out of the red, green, blue and alpha components of a vector", Author = "woei")]
+		#endregion PluginInfo
```
Comment (joreg) [inline vvvv45/addonpack/src/nodes/plugins/Color/RGBVector/RGBVectorNode.cs:44]: help strings should start with a capital letter

Comment (woeishi) [inline vvvv45/addonpack/src/nodes/plugins/Color/RGBVector/RGBVectorNode.cs:44]: hehe, that was a copy paste from the original node,

can change that, if you want

**Comment by tebjan** [comment]
if you do so, please update your branch, as i had to fix a conflict in order to merge it.

**Pull request merged**